flashlight/pkg/CMakeLists.txt

```diff
@@ -11,3 +11,8 @@ include(${FL_PKG_SPEECH_DIR}/CMakeLists.txt)
 
 set(FL_PKG_TEXT_DIR ${FL_PKG_DIR}/text)
 include(${FL_PKG_TEXT_DIR}/CMakeLists.txt)
+
```

Comment:
We need to still have the `cmake_dependent_option` stuff to define `FL_BUILD_PKG_HALIDE` -- this stuff https://github.com/padentomasello/flashlight/pull/8/files#diff-1cb27a80bb5dbadcab78fee60f3bb143b55bd629f01d4426d9af56cd7c3b857bL3-L6

Reply:
`FL_BUILD_PKG_HALIDE`  is defined using `cmake_dependent_option` in https://github.com/padentomasello/flashlight/pull/17/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR285-R288. 

The stuff above is moved into `flashlight/pkg/halide/CmakeLists.txt`

Reply:
I'm not sure I fully understood your comment. Does that resolve it though?

Reply:
Yep - I missed that it was moved.